CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### Fixed
+- Fix the tiff file extension. 
+
```

Comment:
don't forget to bump the version number in the extractor_info.json for geotiff.

Reply:
fixed